wikiclass/feature_lists/wikidatawiki.py

```diff
@@ -76,9 +76,13 @@ def simplify_target(target):
         return target
     elif hasattr(target, "toTimestr"):
         return target.toTimestr()
+    elif isinstance(target, dict) and 'text' in target:
```

Comment:
These switch clauses could use some comments, your reader is left wondering why `target` is such a slippery variable.